auth/grpctransport/grpctransport.go

```diff
@@ -38,7 +38,7 @@ const (
 	// Check env to decide if using google-c2p resolver for DirectPath traffic.
 	enableDirectPathXdsEnvVar = "GOOGLE_CLOUD_ENABLE_DIRECT_PATH_XDS"
 
-	quotaProjectHeaderKey = "X-Goog-User-Project"
+	quotaProjectHeaderKey = "X-goog-user-project"
```

Comment:
So it's OK to leave `X` capitalized here in grpc metadata?

Reply:
Yes, because that is what it is here: https://github.com/googleapis/google-api-go-client/blob/21f10edb1d9d6a3d3baa5b1edcef6588da99def8/transport/grpc/dial.go#L212